ts/features/it-wallet/navigation/ItwRoutes.ts

```diff
@@ -20,6 +20,7 @@ export const ITW_ROUTES = {
   } as const,
   PRESENTATION: {
     PID_DETAILS: "ITW_PRESENTATION_PID_DETAILS",
+    CREDENTIAL_DETAILS: "ITW_CREDENTIAL_DETAILS",
```

Comment:
Given that we want to separate PID from CREDENTIALS, what do you think about adding this screen under CREDENTIALS -> PRESENTATION -> CREDENTIAL_DETAILS?

Reply:
here https://github.com/pagopa/io-app/pull/5219/commits/6d70ba315b9dcf174f7d8604fb35c7a731d2bd2a